recipes/goatools/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-$PYTHON setup.py install
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
 
 binaries="\
```

Comment:
Those scripts should be installed by `setuptools`, hence `build.sh` can be reduced to only
```
 #!/bin/bash
 
$PYTHON setup.py install --single-version-externally-managed --record=record.txt
 ```